esp/templates/survey/questions/checkboxes.html

```diff
@@ -2,10 +2,10 @@
 {% load survey %}
 <table class="fullwidth">
 <tr>
-    <td width="30%"><b>{{ name }}</b></td>
     <td>
+    <b>{{ name }}</b>
     {% for choice in list %}
-        <div style="padding-left:2em; text-indent:-2em;">
+        <div class = "floatLeft">
```

Comment:
I think you might want to preserve the padding and indent here (it was added to implement a hanging indent).

Reply:
Yep, I will add it back.